### DISCUSSION TITLE
Hacktoberfest

### DISCUSSION THREAD

**Discussion by Gaurisha21** [cscodershub/cscodershub-community#133] [General]
Hi 
Can anyone please confirm if this is a valid repository for hacktoberfest? And if yes, then do I need to create an issue first and then pull request or I could directly create a pull request?
Thank you

**Comment by bkayranci** [comment]
> Hi Can anyone please confirm if this is a valid repository for hacktoberfest? And if yes, then do I need to create an issue first and then pull request or I could directly create a pull request? Thank you

You can use following repositories for hacktoberfest,
- [Hacktoberfest Cats](https://github.com/bkayranci/hacktoberfest-cats)
- [Hacktoberfest Dogs](https://github.com/bkayranci/hacktoberfest-dogs)
- [Hacktoberfest Birds](https://github.com/bkayranci/hacktoberfest-birds)
- [Hacktoberfest Rabbits](https://github.com/bkayranci/hacktoberfest-rabbits)
